tesseract_motion_planners/descartes/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ if(TESSERACT_PACKAGE)
     DESCRIPTION "Tesseract motion planners descartes components"
     COMPONENT_DEPENDS core simple
     LINUX_DEPENDS "${TESSERACT_PACKAGE_PREFIX}tesseract-collision-core"
-                  "${TESSERACT_PACKAGE_PREFIX}tesseract-descartes-core"
+                  "${TESSERACT_PACKAGE_PREFIX}descartes-light-core"
     WINDOWS_DEPENDS "${TESSERACT_PACKAGE_PREFIX}tesseract-collision-core"
-                    "${TESSERACT_PACKAGE_PREFIX}tesseract-descartes-core")
+                    "${TESSERACT_PACKAGE_PREFIX}descartes-light-core")
```

Comment:
@marip8 You good with this change. I think this should be good but I think you had added the tesseract prefix.

Reply:
Yes, this was probably a typo